<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OrderManagerForm</class>
 <widget class="QWidget" name="OrderManagerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>523</width>
    <height>376</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Order Info</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="listVerticalLayout">
       <item>
        <widget class="QTreeWidget" name="treeWidget">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <column>
          <property name="text">
           <string>Order Number</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Client Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Product Name</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string># Price</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quantity</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Total Price</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QToolBox" name="toolBox">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="inputPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>259</width>
         <height>316</height>
        </rect>
       </property>
       <attribute name="label">
        <string>&amp;Input</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QFormLayout" name="clientFormLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="idLabel">
            <property name="text">
             <string>&amp;Order Number</string>
            </property>
            <property name="buddy">
             <cstring>idLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="clientLabel">
            <property name="text">
             <string>&amp;Client Name</string>
            </property>
            <property name="buddy">
             <cstring>clientComboBox</cstring>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="quantityLabel">
            <property name="text">
             <string>&amp;Quantity</string>
            </property>
            <property name="buddy">
             <cstring>quantitySpinBox</cstring>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="productLabel">
            <property name="text">
             <string>&amp;Product Name</string>
            </property>
            <property name="buddy">
             <cstring>productComboBox</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="idLineEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="totalPriceLabel">
            <property name="text">
             <string>&amp;Total Price</string>
            </property>
            <property name="buddy">
             <cstring>totalPriceLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="priceLabel">
            <property name="text">
             <string>P&amp;rice per piece</string>
            </property>
            <property name="buddy">
             <cstring>priceLineEdit</cstring>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="priceLineEdit"/>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="totalPriceLineEdit"/>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="clientComboBox">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QComboBox" name="productComboBox">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="quantitySpinBox">
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="clientTreeWidget">
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Phone Number</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Address</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="productTreeWidget">
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Price</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Quantity</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="addHorizontalLayout">
          <item>
           <widget class="QPushButton" name="addPushButton">
            <property name="text">
             <string>&amp;Add</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="modifyPushButton">
            <property name="text">
             <string>&amp;Modify</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="searchPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>259</width>
         <height>316</height>
        </rect>
       </property>
       <attribute name="label">
        <string>&amp;Search</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTreeWidget" name="searchTreeWidget">
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Client Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Produce Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string># Price</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Quantity</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Total Price</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="searchComboBox">
            <item>
             <property name="text">
              <string>ID</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Client Name</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Product Name</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Price</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Quantity</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Total Price</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="searchLineEdit"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="searchPushButton">
          <property name="text">
           <string>S&amp;earch</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
